packages/config/src/configs/moonbaseAlpha.ts

```diff
@@ -306,6 +307,32 @@ export const moonbaseAlphaConfig = new ChainConfig({
         balance: BalanceBuilder().substrate().system().account(),
       },
     }),
+    new AssetConfig({
+      asset: dev,
+      balance: BalanceBuilder().substrate().system().account(),
+      contract: ContractBuilder().Xtokens().transfer(),
+      destination: peaqEvmAlphanet,
+      destinationFee: {
+        amount: 0.00000001,
```

Comment:
do floating point numbers work well or can we get some weird behavior like unwanted rounding or issues during math operations?

Reply:
This doesn't matter for now and will be fixed later.